Source/Core/Core/IOS/USB/USB_HID/HIDv4.cpp

```diff
@@ -103,7 +103,7 @@ std::optional<IPCReply> USB_HIDv4::GetDeviceChange(const IOCtlRequest& request)
 IPCReply USB_HIDv4::Shutdown(const IOCtlRequest& request)
 {
   std::lock_guard lk{m_devicechange_hook_address_mutex};
-  if (m_devicechange_hook_request != 0)
+  if (m_devicechange_hook_request)
```

Comment:
Just wondering, you remove the `!= 0` here but left it as `!= nullptr` in `OGLTexture.cpp` - it feels like we should prefer one of the two for consistency.

Do note that I did _not_ look at the rest of the file to see if it was a per-file-consistency thing. If thats the case, ignore this remark.

Reply:
I left it at this because this is a std::unique_ptr, which has a implicit conversion to bool.  I was on the fence about this change myself.  Maybe it would be better to compare against nullptr.